README.adoc

```diff
@@ -89,8 +90,8 @@ mvn verify
 {: codeblock}
 endif::[]
 
-This command might take some time to run the first time because the dependencies and the Docker image for Open Liberty must download. If you 
-run the same command again, it will be faster.
+This command might take some time to run the first time because the dependencies and the Docker image for Open Liberty must download. 
+If you run the same command again, it will be faster.
 
 The previous example shows how you can run integration tests from a cold start. With Open Liberty development mode, you can use MicroShed Testing to run tests on
 an already running Open Liberty server. Run the following Maven goal to start Open Liberty in development mode:
```

Comment:
Change "on an already running Open Liberty server" to "on an active Open Liberty server" to be more concise.